Nos/Service/RelayService.swift

```diff
@@ -284,6 +281,7 @@ extension RelayService {
             let allSubscriptions = await subscriptions.all
             let fulfilledSubscriptions = try await self.backgroundContext.perform {
                 let relay = self.relay(from: socket, in: self.backgroundContext)
+                // TODO: we are letting in duplicate events somehow
```

Comment:
Is there a ticket for this?

Reply:
There is now! https://github.com/planetary-social/nos/issues/254